keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -245,20 +217,24 @@ def call(self, images, training=False):
 
         return box_pred, cls_pred
 
+    def call(self, images, training=None):
+        box_pred, cls_pred = self._forward(images, training=training)
+        if not training:
```

Comment:
I'm not certain how this will work with functional subclassing -- it seems like we'll want to get rid of `_forward` in the functional subclass model (that's how I've done it in my implementation of RetinaNet as a functional subclass -- PR pending #1203 being merged)

I suppose we can override call in the functional subclass model to call super.call and then decode the deltas to real boxes?

Reply:
Discussed offline, let's try this in your PR